### ISSUE TITLE
Switch to turn off population frequency plot

### ISSUE THREAD

**Issue by savitakartik** [tskit-dev/tsqc#144]
We can now view population frequencies of a mutation by clicking on a data point in the main mutations view. For this, we're computing a numpy array of shape num_nodes x num_populations. With large number of populations (e.g. >200 in  the Unified Genealogies dataset) this array grows extremely big and tsqc runs out of memory, getting killed. We should provide an option to turn off computing population data. This way users can turn on the switch to view frequencies only when sufficient memory is available.

**Comment by benjeffery** [comment]
Can we calculate these on-the-fly instead?

**Comment by jeromekelleher** [comment]
Impractical at chromosome scale I'm afraid. Possible for a small window, though?

**Comment by benjeffery** [comment]
Don't you just need to do it for the single mutation that's been clicked on?

**Comment by savitakartik** [comment]
Yes, it's only for a the single mutation that's clicked on. We considered on-the-fly calculations, but I think one of the reasons we thought it would be easier to pass the whole set of arrays to the browser, was so interactivity is smoother for larger datasets. 

The reason for creating this issue was that a big machine was needed everytime we served the dataset, even when we care about the population frequencies. Now that we have the preprocess logic, perhaps this issue is not as relevant? I'll do a couple of checks and confirm.

**Comment by jeromekelleher** [comment]
If we're doing it for a single mutation then maybe it is feasible. You'd have to do this:
```python
tree = ts.at(mutation_position)
nodes =  tree.preorder(mutation_node) # numpy array
subtree_population = ts.nodes_population[nodes]
# return a data frame of the counts
```

**Comment by benjeffery** [comment]
Moving this to MVP+1 as it isn't a showstopper and we have a good solution.
